documentation/modules/post/windows/gather/credentials/whatsupgold_credential_dump.md

```diff
@@ -0,0 +1,305 @@
+## Vulnerable Application
+
+This module exports and decrypts credentials from WhatsUp Gold to a CSV file; it is intended as a
+post-exploitation module for Windows hosts with WhatsUp Gold installed. The module has been tested
+on and can successfully decrypt credentials from WhatsUp versions 11.0 to the latest (22.x).
+Extracted credentials are automatically added to loot.
+
+## Actions
+
+### Dump
+
+`dump` is the default action and performs extraction of the WhatsUp Gold database parameters and
+encryption keys. This action also exports WhatsUp Gold SQL data and immediately decrypts it. `dump`
+is suitable when the following conditions are met:
+
+1. The sqlcmd binary is available on the target system
```

Comment:
It may be worth adding a `check` method to determine if the binary is present.

Reply:
My understanding is that post modules do not support check, is there a different approach I should take?